### ISSUE TITLE
App should display a welcome message on first usage

### ISSUE THREAD

**Issue by dandv** [xor-gate/syncthing-macosx#188]
I've just installed v1.23.0 on Ventura 13.1 (Apple M1 Max). When I open `Syncthing.app`, nothing appears to happen. This can be disconcerting to new users until they notice the new icon among the status menus.

**Comment by joelowjw** [comment]
not sure what's happening. I downloaded and installed the latest version and tried to run it. But nothing happened or popped up.

**Comment by xor-gate** [comment]
> not sure what's happening. I downloaded and installed the latest version and tried to run it. But nothing happened or popped up.

I must say, I never seen this kind of problem. Could you check if syncthing is started and if its logfile is created:
~/Library/Application Support/Syncthing/syncthing.log, use Finder -> Go -> Go to Folder to open it. My assumption it is not created as the macOS application runs/manages the syncthing service (instead of macOS launchd). 

Could you be a little more specific about which OS version you are running? It sounds odd but does a reboot help (normally macOS needs not to rebooted for long time in my experience).

**Comment by joelowjw** [comment]
Yes, the logfile is created.
I'm on OS 13.2.1

**Comment by xor-gate** [comment]
> Yes, the logfile is created. I'm on OS 13.2.1

I would really not expect you have a logfile. That means it does actually run. 
So you have no tray icon in the status bar or whatsoever? 

<img width="509" alt="Screenshot 2023-03-24 at 21 19 17" src="https://user-images.githubusercontent.com/1050166/227630493-fe050849-9b8d-464f-bfec-01997641feca.png">

**Comment by joelowjw** [comment]
yea nothing pops up in the status bar... and when i tried to click on the app to run, it says that it is already running

**Comment by xor-gate** [comment]
@joelowjw I have created a new issue #192 as this is very different from the feature request of the original ticket owner. Please read the information in the new ticket, and comment there. For now I'm unable to resolve this problem. My suggestion would be to run syncthing automatically with launchd as a macOS service and use the webbrowser to access it. I know this is completely not user friendly, and that is the case why `syncthing-macos` was created. Sorry!

**Comment by joelowjw** [comment]
No worries. Thank you so much for trying to troubleshoot it.
Appreciate it!

**Comment by xor-gate** [comment]
@joelowjw, see ticket #192. On the forum some other people have tested and confirmed macOS 12.3.1 should work.

**Comment by LilithHafner** [comment]
I came here to report "nothing happens when I run it" and only after reading this issue did I realize that I do have an icon in my tray. However, I didn't notice it. Until I saw this ticket, my plan was to report the issue and give up.

I recommend giving this a priority higher than "Unplanned"
